python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -57,6 +70,12 @@ class devnull:
             self.app.config["server"] = server
             server.serve_forever()
 
+            # Closes server on program exit, including freeing all sockets
+            def closeserver():
+                server.stop()
+
+            atexit.register(closeserver)
```

Comment:
when is atexit called back? The idea for the server is to create one lazily and then keep it to serve all later calls to the dashboard. This would enable multi-explanation views in the future and allow us to reduce the fetched js (this reduction in fetching from cdn has not been implemented yet, there is a task for it in the backlog, but it can be done when there are free cycles).

Reply:
atexit is called when python process exits, so I think it should be fine.  I just wanted to clean up correctly.